src/main/java/dinkplugin/notifiers/data/GroupStorageNotificationData.java

```diff
@@ -17,13 +17,16 @@ public class GroupStorageNotificationData extends NotificationData {
     Collection<SerializedItemStack> withdrawals;
     long netValue;
     String groupName;
+    boolean includePrice;
```

Comment:
```suggestion
    transient boolean includePrice;
```

we can exclude this field from json serialization - alternatively we could convert `netValue` into `Long` so it can be `null`